### DISCUSSION TITLE
Why sealed segment is not detect?

### DISCUSSION THREAD

**Discussion by hjlee9182** [milvus-io/milvus#30543] [Q&A and General discussion] [answered]
Hi, I'm newbie about milvus.
I insert in 2 million data. There's a size limit every time I insert, so I think I did 20,000 data about 100 times

1. Why no sealed segments in metric? I guess when I insert data, Growing -> sealed -> flushed segment, but sealed can't see
2. Why is there a dropped segment? Because I think the dropped segment only occurs when you delete it

![image](https://github.com/milvus-io/milvus/assets/46337218/232cc6cf-5c37-466a-b6d8-02f58fbf65ec)

**Comment by yhmo** [comment, marked as answer]
In this metric, flushed is equal to sealed.
The dropped segments are from compaction. 
A compaction operation for segment_A and segment_B. The two segments are combined into a new segment_C, then the segment_A and segment_B will be marked as "dropped", and waiting for GC operation. In other words, segment_C replaces segment_A and B.
